content/_index.md

```diff
@@ -4,27 +4,66 @@ date: 2019-06-28
-Packit project is composed of two main components:
+### What is Packit?
 
-1. [packit]({{< ref "cli" >}}), a CLI tool, which you can
+1. a [CLI tool]({{< ref "cli" >}}), which you can
    [install locally](/docs/guide/#have-packit-tooling-installed-locally)
-   and start using right away.
+   and start using right away,
 
-2. [Packit Service]({{< ref "packit-service" >}}), can provide you feedback
-   on how is your project integrated with Fedora Linux or CentOS Stream.
+2. a [service]({{< ref "packit-service" >}}), which provides you feedback
+   on how is your project integrated with [Fedora Linux](https://getfedora.org), [CentOS Stream](https://www.centos.org/centos-stream/)
+   and other distributions.
+   <--->
 
-Feel free to jump into [the guide]({{< ref "guide" >}}) for using packit.
+### Who should use Packit?
 
-### Contact
+Upstream maintainers who would like to automate validation, testing and releasing of code changes
+in downstream distributions.
+{{< /columns >}}
 
-If you encounter a problem while using packit or Packit Service, please open an
-[upstream issue](https://github.com/packit/packit-service/issues/new).
-In case of any other questions, feel free to contact
-[us](https://github.com/orgs/packit/teams/the-packit-team):
+### What key features does Packit offer?
 
-- **#packit** on [Libera.Chat](https://libera.chat/)
-- **hello@packit.dev**
+{{< columns >}}
+
+#### Building RPMs
+
+Packit uses [Copr build system](https://copr.fedorainfracloud.org/) to build RPMs from your code changes,
+so you can validate your package will be correctly built from each pull request or have automation around your Copr
+repository containing builds from your upstream releases/commits.
+<--->
+
+#### Testing
+
+Packit runs your tests in [Testing Farm](https://docs.testing-farm.io/general/0.1/index.html) and can test either the RPM package built in Copr or
+trigger tests directly on different platforms.
+<--->
+
+#### Automating Fedora releases
+
+With Packit, you can easily
+get your upstream releases to the [Fedora dist-git](https://src.fedoraproject.org), automatically submit builds in [Koji](https://koji.fedoraproject.org/koji/), and we are currently
+finalizing implementation for automatic creation of [Bodhi](https://bodhi.fedoraproject.org) updates!
+{{< /columns >}}
+
+### How does Packit look like in action?
+
+![Packit workflow](/images/build_test_workflow.jpg)
+
+### Why start using Packit?
+
+1. _Easy to configure_ - Feel free to jump into [our guide]({{< ref "guide" >}}) or check directly the [configuration documentation](/docs/configuration).
+
+2. _Stable users_ - [Systemd](https://github.com/systemd/systemd/), [cockpit](https://github.com/cockpit-project/cockpit), [avocado](https://github.com/avocado-framework/avocado), [ansible-lint](https://github.com/ansible-community/ansible-lint) and many other projects. If you are interested,
```

Comment:
Hmm, should we link the main page of these projects - or should we link directly to packit.yaml in their git repo?

Reply:
I was also thinking about linking to packit.yaml but... we'd need to make sure they don't rename it or move around (or even drop 😩), so the conclusion in my head was to link to the github project and let people "discover" how they use packit (open a PR, click on checks, look at packit.yaml)

Reply:
We could still use permalinks to the current versions of their packit configs, but that could get outdated as well, so I am also not sure :confused:

Reply:
maybe it's finally time to start checking URLs in our docs if they are up to date :D

anyway, I'm +1 to link directly the configs (since that may be the first thing people look for)

Reply:
> We could still use permalinks to the current versions of their packit configs, but that could get outdated as well, so I am also not sure confused

We can link to specific treeish. E.g.,  https://github.com/systemd/systemd/blob/d15e1a29e3aab04ee79d5e3ec8e1e65fca78e165/.packit.yml But this, on the other hand, does not get updates when we update some of our API (format of packit.yaml), which will be incompatible with the current format. Hmm, but that may or may not happen. Definitely in the distant future. :)

Reply:
> We can link to specific treeish. E.g., https://github.com/systemd/systemd/blob/d15e1a29e3aab04ee79d5e3ec8e1e65fca78e165/.packit.yml 

Yes, that's what I meant by permalinks.

Agreed, changed it to the links to configs.